### PR TITLE
Feat lists

### PR DESCRIPTION
**Pull request by ankit9015**
Lists feature includes:
1. Ordered lists of different styles
2. Unordered lists of different styles
3.  Stacked List

https://user-images.githubusercontent.com/67504902/154798277-33743dfb-d490-433f-a748-449b37f78ed1.mp4

### CODE REVIEW

**Review by psychoactiv — APPROVED** [review]
code looks good

**Review by Kothalkarpriya — APPROVED** [review]
Good work, you can merge

**Comment by Kothalkarpriya** [comment]
Do check what conflicts this branch has, first resolve them before going for reviews, It would be better

File: components/lists/lists.css [hunk 1]
```diff
@@ -0,0 +1,76 @@
+html {
+
```
Comment (ajit-sharmaGH) [inline components/lists/lists.css:2]: remove, if it's not in use

**Review by ajit-sharmaGH — APPROVED** [review]
try to resolve the conflicts!

**Pull request merged**